elasticdl_preprocessing/tests/to_ragged_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import tensorflow as tf
+
+from elasticdl_preprocessing.layers.to_ragged import ToRagged
+from elasticdl_preprocessing.tests.test_utils import ragged_tensor_equal
+
+
+class ToRaggedTest(unittest.TestCase):
+    def test_dense_to_ragged(self):
+        layer = ToRagged()
+        inp = tf.constant([[1], [-1], [4]], tf.int64)
```

Comment:
Use abbreviations when they are commonly used, otherwise, you are making it hard to read the code. inp for input?